.goreleaser.yml

```
@@ -0,0 +1,39 @@
+# Port flags from Makefile.
+builds:
+- env:
+  - CGO_ENABLED=0
+  - GO111MODULE=on
+  ldflags:
+  - -X github.com/kinvolk/lokomotive/pkg/version.Version={{.Version}}
+  - -extldflags '-static'
+  flags:
+  - -buildmode=exe
+  goarch:
+  - amd64
+  goos:
+  - linux
+  - darwin
+  main: ./cmd/lokoctl/main.go
+  binary: lokoctl
+
+changelog:
+  # Skip changelog, we'll generate it semi-manually and pass it to goreleaser.
+  skip: true
+
+# Enable signing checksums.
+signs:
+  - artifacts: all
```

```suggestion
- artifacts: all
```

We should be consistent with lists indentation.